.gitignore

```diff
@@ -115,3 +115,8 @@ SECRET
 # substra-backend files
 docker/docker-compose-dynamic.yaml
 **/medias/*
+
+
+# charts tmp files
+charts/substra-backend/tmpcharts/*
+skaffold-overrides.yaml
```

Comment:
Can you provide more information on `skaffold-overrides.yaml`, it's not something related to the description of the PR and the file already available in the repo ?

Reply:
sure, I updated the PR description.